CONTRIBUTING.md

```diff
@@ -18,6 +18,9 @@ We use a four step process for contributions:
 1. Create a GitHub Pull Request for your change, following the instructions in the pull request template.
 1. Perform a Code Review with the project maintainers on the pull request.
 
+Remember to also run the full build (`./mvnw clean package -DskipTests`) before
+you open the pull request to format your changes.
```

Comment:
Thank you for pushing this commit in (and the change to the GitHub action - that's very helpful too, thank you). Personally I would prefer the wording to fit more into the list above (in particular I would mention that the way it is parseable at this time doesn't technically direct the contributor to include the formatted code within their pull request and committing takes place in step 2) but in my opinion, updating the CONTRIBUTING.md so as to be able to identify to a contributor that code formatting is a part of the process is the critical thing to introduce with this pull request and that has been done. marcosgopen is should still provide the review, but I would share that it  works for me.

Thank you, Martin!